Adds all friendly name audit work to the branch

victoriaatraft opened the pull request:
Adds all friendly name audit work to the branch

@ADPennington, @reitermb and I wanted to bring your attention to the items below. Specifically 59 in the M2.py file as there were some inconsistencies with the code base and the instructions. 
Miles, please feel free to add if I missed anything.

victoriaatraft committed:
Adds all friendly name audit work to the branch

Adds all friendly name audit work to the branch

victoriaatraft commented on tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py:
```diff
@@ -270,7 +270,7 @@
                 Field(
                     item="32A",
                     name='FED_OASDI_PROGRAM',
-                    friendly_name="federal old-age survivors and disability insurance program",
```
The codebook's reference is: Receives Federal Disability Insurance Benefits Under Title II of the Social Security Act (Social Security Disability Insurance [SSDI]).

victoriaatraft commented on tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py:
```diff
@@ -300,7 +300,7 @@
                     item="32D",
                     name='AID_AGED_BLIND',
-                    friendly_name="receives from aid to the aged, blind, and disabled program",
+                    friendly_name="Receives Disability Benefits: Aged, Blind, and Disabled Under Title XVI-AABD",
```
The codebook says: Code no longer in use, Enter 0

victoriaatraft commented on tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py:
```diff
@@ -750,7 +750,7 @@
                 Field(
```
59 in particular did not align with the original coding instructions and wanted to bring that to your attention. Comparing to the code book, is no section for E and there are no two A's. Additionally what was in the python file for names, does not correlate with the instructions. I've attached an image for a visual. 

There are similar instances with other files that are [documented in the excel sheet here](https://gorafttech-my.sharepoint.com/:x:/g/personal/vamoroso_teamraft_com/EbOyrZPqlA1Chz8oaVTWDKQBj5CmOYb-N8-U606f1gMvQg?e=TjdlOp).

![Amount of Unearned Income m2](https://github.com/raft-tech/TANF-app/assets/106103383/37c9c0a0-8a12-4a84-baaf-8470be2330b5)

reitermb reviewed (approved):
Victoria and I synced up 3/27 to overview—looks great from my perspective, just awaiting pairing to walk through design choices for the wordier edge cases with @ADPennington currently planned for UX sync on 4/2

🚀 🙌

ADPennington commented on tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py:
```diff
@@ -150,7 +150,7 @@
                 Field(
                     item="14",
                     name="BLANK",
```
@elipe17 this `name` here should be `FAMILY_AFFILIATION`. can we change this here or will this introduce problems? cc: @victoriaatraft

reitermb commented on tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py:
```diff
@@ -448,8 +448,7 @@
             item="67",
             name="DATE_OF_BIRTH",
-            friendly_name="date of birth",
-            type="string",
```
@victoriaatraft looks like the type= line got dropped here, can you add `type="string",` back in alongside the other final tweaks? That should resolve the build errors.

reitermb commented on tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py:
```diff
@@ -557,7 +557,7 @@
             item="65B",
             name='RECEIVE_SSI',
-            friendly_name="receives ssi",
+            friendly_name="Receives Disability Benefits: Supplemental Security Income Under Title XVI-SSI or Aged, Blind, and Disabled Under Title XVI-AABD ",
```
@victoriaatraft one more little tweak, could we remove the extra space at the end of this?

ADPennington commented:
back in raft review until ready for qasp @victoriaatraft @reitermb

victoriaatraft committed:
Friendly names update 2

Removed spacing on m3.py
Removed months on t6.py, m6.py
Code no longer in use for m1.py, t1.py
Text updates to t2.py
Type from number to string t3.py

victoriaatraft committed:
Additional change

fixed typo

victoriaatraft committed:
Ethnicity/Race Updates

Including Ethnicity/Race:

reitermb committed:
Revert "Merge branch 'develop' into friendlynamefixes"

This reverts commit 0da1a0f5aa0818a7f96120f9f5e879575dffc47d, reversing
changes made to 00dbe5e9b229b4e99a7154d99198a5673f318636.

elipe17 committed:
- Update name
- Fix test

elipe17 committed:
Merge branch 'develop' of https://github.com/raft-tech/TANF-app into friendlynamefixes

elipe17 committed:
- Fix failing test

reitermb commented:
Closing this PR, changes moved to #2931
